Pod/Classes/SEGUrbanAirshipIntegration.m

```diff
@@ -76,19 +75,7 @@ - (void)identify:(SEGIdentifyPayload *)payload {
 }
 
 - (void)screen:(SEGScreenPayload *)payload {
-    NSString *screen = kUrbanAirshipScreenPrefix;
-
-    if (payload.category) {
-        [screen stringByAppendingString:[NSString stringWithFormat:@"_%@", payload.category]];
-    }
-
-    if (payload.name) {
-        [screen stringByAppendingString:[NSString stringWithFormat:@"_%@", payload.name]];
-    }
-
-    [[[UAirship shared] analytics] trackScreen:screen];
-
-    [self addEvent:screen properties:payload.properties];
+    [[[UAirship shared] analytics] trackScreen:(payload.name ?: payload.category)];
```

Comment:
what's the thinking behind this bit where we prefer name and fall back on category? I ask partly because I don't really know what SEGScreenPayload is used for.